virttest/test_setup.py

```diff
@@ -2225,9 +2225,9 @@ def remote_session(params):
     :param params: Test params dict for remote machine login details
     :return: remote session object
     """
-    server_ip = params["server_ip"]
-    server_user = params.get("server_user", "root")
-    server_pwd = params["server_pwd"]
+    server_ip = params["remote_ip"]
```

Comment:
we use remote_ip, remote_pwd and remote_user in migration or in remote uri testcases for target credentials. But this `remote_session()` can be used as wrapper in a generic way to take sessions for any host/guest machine, so we can avoid conflict of params.